mmv1/api/type.rb

```diff
@@ -121,6 +132,8 @@ module Fields
       # if true, then we get the default value from the Google API if no value
       # is set in the terraform configuration for this field.
       # It translates to setting the field to Computed & Optional in the schema.
+      # For nested fields, this also needs to be set on each ancestor (ie. self,
```

Comment:
Ah- importantly this _applies at a single level_. It's both valid to have a non-O+C field with an O+C child (such as a complex default within a block) and to have an O+C field with a non-O+C child (a required subfield when the block is specified)

Reply:
You need to set it through the tree to make the whole tree O+C, which you generally don't need to do.

Reply:
Ah ok, I think I just had this wrong then. I've updated, let me know if the wording I chose makes sense.